react-backend/client/src/components/admin/adminUsersPage.js

```diff
@@ -0,0 +1,297 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { Button } from '../visitors/button';
+import { Logoutbutton } from '../visitors/logoutbutton';
+
+export class AdminUsersPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: [],
+      reauthenticated: false,
+      failure: false,
+      password: '',
+      activities: [],
+      filters: [],
+      orderBy: '',
+    };
+  }
+
+  headers = new Headers({
+    Authorization: localStorage.getItem('token'),
+    'Content-Type': 'application/json',
+  });
+
+  handleChange = e => {
```

Comment:
I know there are a few of these in other files, but we might as well fix the new ones early:

```js
  this.setState({ [e.target.name]: e.target.value });
```